dashboard/app/controllers/openai_evaluate_controller.rb

```diff
@@ -52,6 +52,15 @@ def evaluate
     elsif ProfanityFilter.find_potential_profanity(student_work, "en", {})
       json_response = {"content" => profanity_detected_response.to_json}
       return render(status: :ok, json: json_response)
+    elsif Rails.env.test?
```

Comment:
I tested this locally by adding `|| Rails.env.development?` here.  I didn't want to leave that in though as we actually test for acuracy on our local environments.